test/recipes/25-test_req.t

```diff
@@ -355,6 +355,47 @@ subtest "generating SM2 certificate requests" => sub {
     }
 };
 
+subtest "generating certificate requests with -cipher flag" => sub {
+    plan tests => 4;
+
+    SKIP: {
+        skip "RSA is not supported by this OpenSSL build", 2
+            if disabled("rsa");
+
+        diag("Testing -cipher flag with aes256...");
+
+        ok(run(app(["openssl", "req",
+                    "-config", srctop_file("test", "test.cnf"),
+                    "-new", "-out", "testreq-rsa-cipher.pem", "-utf8",
+                    "-key", srctop_file("test", "testrsa.pem"),
+                    "-cipher", "aes256"])),
```

Comment:
You need to use `-newkey` and `-keyout` like you did in the documentation. Also please change `aes256` to `aes-256-cbc`